server/app/services/applicant/question/Question.java

```diff
@@ -121,6 +121,10 @@ public String getDefaultAnswerString() {
     return "-";
   }
 
+  public Path getContextualizedPath() {
```

Comment:
This is not currently tested directly, but lmk if you think we should even though it's just passing through the value form the internal ApplicantQuestion